datameta/api/openapi.yaml

```diff
@@ -321,8 +321,12 @@ paths:
         - cookieAuth: []
         - {} # Unauthenticated access for use with reset token
       responses:
-        "204":
+        "200":
```

Comment:
Sorry, didn't find a way to comment in an unchanged file. As discussed, the response code should also be changed in the frontend: `account.js` line 28